src/developer/web-api/metadata.md

```diff
@@ -323,6 +324,10 @@ Get data elements with id property ID1 or ID2:
 
     /api/dataElements?filter=id:eq:ID1&filter=id:eq:ID2
 
+Get data elements, ignoring case, with name property Data Element One:
+
+    /api/dataElements?filter=name:ieq:data element one
```

Comment:
I think spaces in an URL is not a good example as they are not allow unnecessarily complicating the example with a problem that is not about what this wants to demonstrate.

Reply:
updated